acceptance/tests/config-entries/config_entries_test.go

```diff
@@ -331,6 +376,9 @@ func TestController(t *testing.T) {
 				logger.Log(t, "deleting terminating-gateway custom resource")
 				k8s.RunKubectl(t, ctx.KubectlOptions(t), "delete", "terminatinggateway", "terminating-gateway")
 
+				logger.Log(t, "deleting control-plane-request-limit custom resource")
+				k8s.RunKubectl(t, ctx.KubectlOptions(t), "delete", "controlplanerequestlimit", "control-plane-request-limit")
+
```

Comment:
This is missing an assertion that the resource has been deleted. Additionally, these steps need to be copied over to the namespaces test. you can copy paste them for the most part but you will need to add some `-n kubens` flags here and there. it should be clear in the test itself.